fish_prompt.fish

```diff
@@ -665,6 +669,15 @@ function __bobthefish_prompt_user -S -d 'Display current user and hostname'
         echo -ns (whoami)
     end
 
+    if set -q display_sudo_user
+        if set -q display_user
+            echo -ns ' '
+        else
+            __bobthefish_start_segment $color_username
+        end
+        echo -ns "($SUDO_USER)"
+    end
+
     if set -q display_hostname
         if set -q display_user
```

Comment:
Oh, I missed this. It will need to be updated to not start a new segment if _either_ `display_sudo_user` or `display_user` is true.

Reply:
Oh, you're totally right! I missed that, too.  
Fixed that now!